server/routes/server.routes.js

```diff
@@ -57,7 +57,15 @@ router.get('/verify', (req, res) => {
 });
 
 router.get('/sketches', (req, res) => {
-  res.send(renderIndex());
+  req.user ? res.send(renderIndex()) : get404Sketch(html => res.send(html));
+});
+
+router.get('/assets', (req, res) => {
+  req.user ? res.send(renderIndex()) : get404Sketch(html => res.send(html));
```

Comment:
After using this, I think it's confusing that this renders a 404 page if the user is not logged in—I think this should redirect to `/login`.